modules/serverless-rn-1-13-0.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-13-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} 1.13.0
+
+[id="new-features-1-13-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.19.0.
+* {ServerlessProductName} uses Knative Eventing 0.19.2.
+* {ServerlessProductName} now uses Kourier 0.19.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.19.1.
+* {ServerlessProductName} now uses Knative Kafka 0.19.1.
+* A `DomainMapping` custom resource (CR) has been added to {ServerlessProductName} to enable users to map a custom domain name to a Knative Service. See the Knative documentation on https://knative.dev/docs/serving/creating-domain-mappings/[Creating a mapping between a custom domain name and a Knative Service].
+* In Knative Serving 0.19.0, `v1alpha1` and `v1beta1` versions of the `Service`, `Route`, `Configuration`, and `Revision` resources have been deprecated. The {ServerlessOperatorName} automatically upgrades these resources to `v1`, so no user action is required.
```

Comment:
The resources were already deprecated in 0.18.x. Now they're completely removed. We need to fix this.